node/Makefile

```diff
@@ -529,7 +538,7 @@ st: $(REMOTE_DEPS) image dist/calicoctl busybox.tar calico-node.tar workload.tar
 # CI/CD
 ###############################################################################
 .PHONY: ci
-ci: static-checks fv image build-windows-archive st
+ci: static-checks ut fv image build-windows-archive st
```

Comment:
how long do the UTs take to run?

Reply:
They won't add much at all to the pipeline time, it's just a few tests.
```
Ginkgo ran 2 suites in 1.838781692s
Test Suite Passed

real    0m2.230s
user    0m0.078s
sys     0m0.025s
```